test/scala/com/github/johnynek/bazel_deps/ModelTest.scala

```diff
@@ -100,4 +100,17 @@ class ModelTest extends FunSuite {
     // special: remove default packaging
     assert(MavenArtifactId("foo:jar").addSuffix("_1").asString == "foo_1")
   }
+
+  test("MavenCoordinate toXml"){
+
+    val mavenCoordinate = new MavenCoordinate(MavenGroup("com.twitter"), MavenArtifactId("finagle"), Version("0.1"))
```

Comment:
can we add a test of an entire file where we parse some yaml and then emit the pom?